common/src/main/scala/org/apache/celeborn/common/util/Utils.scala

```diff
@@ -1072,4 +1074,20 @@ object Utils extends Logging {
     }
     labelPart(0).trim -> labelPart(1).trim
   }
+
+  def getHadoopFS(conf: CelebornConf): FileSystem = {
```

Comment:
How about moving it to `CelebornHadoopUtils`? and we should use `CelebornHadoopUtils#newConfiguration` instead of `new Configuration()`

Reply:
Here needs an empty configuration so that celeborn conf can override pre-defined settings. So It can not be moved to CelebornHadoopUtils.

Reply:
I see, the configuration priority here is 

1. core-site.xml, hdfs-site.xml
2. hardcoded
3. CelebornConf

It changes the current behavior but looks reasonable. I'm wondering if we can change the `CelebornHadoopUtils#newConfiguration` and pass a Map to achieve this ability.

Reply:
And, we'd better to document this behavior in some place

Reply:
Comments addressed.